When make install the dragonfly, the log show:cannot stat ‘../bin/linux_amd64/dfdaemon’: No such file or directory

## Question
<!-- You can ask any question about this project -->
make build-supernode && make install
......
Begin to install dfget and dfdaemon and supernode.
./hack/install.sh install
install: /opt/dragonfly/df-client
cp: cannot stat ‘../bin/linux_amd64/dfdaemon’: No such file or directory
make: *** [install] Error 1

--- a/docs/user_guide/install_server.md
+++ b/docs/user_guide/install_server.md
@@ -96,7 +96,7 @@ docker run -d --name supernode --restart=always -p 8001:8001 -p 8002:8002 -v /ho
 3. Compile the source code.
 
     ```sh
-    make build-supernode && make install
+    make build-supernode && make install-supernode
     ```
 
 4. Start the SuperNode.

